Comments/ClientApp/src/components/Review/Review.js

```diff
@@ -72,7 +73,7 @@ export class Review extends Component<PropsType, StateType> {
 			loading: true,
 			consultationData: null,
 			commentsData: null,
-			userHasSubmitted: false,
+			submittedDate: false,
```

Comment:
Should this be null?

Reply:
Has been changed to null as per suggestion.